tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -271,10 +294,53 @@ private void CheckUnboundProbes()
             }
         }
 
-        private void AcceptConfiguration(ProductConfigChangedEventArgs args)
+        private void AcceptAddedConfiguration(ProductConfigChangedEventArgs args)
+        {
+            var snapshots = new List<SnapshotProbe>();
+            var metrics = new List<MetricProbe>();
+            ServiceConfiguration serviceConfig = null;
+
+            foreach (var configContent in args.ConfigContents)
```

Comment:
We should add a try/catch within this `foreach` loop such that if we fail to deserialize or apply one of the added probes, it will fail with an `ERROR` Probe Status, but won't cause us to skip the succeeding probes.

Reply:
As discussed, I've surrounded error handling policy with `try..catch` here e223e45038dfe3403547a837c77b4815f535e357